styles.css

```diff
@@ -0,0 +1,101 @@
+:root {
+  --font-roboto: "Roboto", sans-serif;
+  --font-crete-round: "Crete Round", serif;
+  --font-poppins: "Poppins", sans-serif;
+  --color-primary: #1c1a19;
+  --color-secondary: #ff6b00;
+  --color-white: white;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+.navbar {
+  background-color: #3c3a39;
+  padding: 2em;
+}
+
+.navbar-wrapper {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.my-logo {
+  color: var(--color-white);
+  font-family: var(--font-roboto);
+  font-size: 2em;
+  font-weight: 200;
+}
+
+.hamburger-menu {
+  width: 3.5em;
+  filter: invert(100%);
+}
+
+.headline {
+  background-color: var(--color-primary);
+  width: 100%;
+  height: 59em;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+}
+
+.headline-wrapper {
+  padding: 2em;
+}
+
+.greeting-title {
+  color: #ff6b00;
+  font-size: 5.6em;
+  font-family: "Crete Round", serif;
+  font-weight: 400;
+}
+
+.greeting-description1 {
+  color: var(--color-white);
+  font-family: var(--font-roboto);
+  font-weight: 600;
+  font-size: 2.4em;
+  margin-top: 0.5em;
+  margin-bottom: 0.5em;
+}
+
+.greeting-description2 {
+  color: var(--color-white);
+  font-family: var(--font-poppins);
+  font-size: 1.6em;
+  font-style: normal;
+  line-height: 1.5em;
+}
+
+.headline-social-media-links {
+  display: flex;
+  align-items: center;
+}
+
+.social-media-link li {
+  list-style-type: none;
+}
+
+.social-media-link {
+  margin-right: 1em;
+}
+
+.illustration {
+  position: absolute;
+  right: -280;
+  bottom: 0;
+  rotate: -17deg;
+}
```

Comment:
- [ ] Please make sure your background image does not overlap the body and overflow-x is not possible. That would increase the user experience.
![image](https://user-images.githubusercontent.com/91486791/235143498-b9d04236-7266-4f30-8c6d-55006095ae9f.png)